keanu-python/setup.py

```diff
@@ -7,12 +7,13 @@ def readme():
 setup(name='keanu',
       description='A probabilistic approach from an Improbabilistic company',
       long_description=readme(),
-      version='0.0.14.dev0',
+      version='0.0.14.dev8',
```

Comment:
`dev0`?

Reply:
I'm just incrementing this every time I upload a version to pypi - because it fails if the version already exists.